### ISSUE TITLE
Feature Request: Toggle Day/Month/Year BTC and equivilant currency

### ISSUE THREAD

**Issue by billmcilhargey** [nicehash/NiceHashMiner2#92]
Allow a toggable (or maybe hover over the amount in the details and provide a tooltip) of values for day/week/month profit

**Comment by S74nk0** [comment]
@computeronix Are you referring to extending the daily estimated earnings control? 
![dee_ctrl](https://user-images.githubusercontent.com/1517233/27336185-119267e6-55cf-11e7-9a26-a569d4c6cfaa.PNG)

**Comment by billmcilhargey** [comment]
Yes, having a tool tip or even if you click it, let it cycle between Daily, Weekly, Monthly values :)
